mapkit/mapkit.Annotation.d.ts

```diff
@@ -380,12 +380,5 @@ declare namespace mapkit {
     titleVisibility?: string;
   }
 
-  // prettier-ignore
-  interface AnnotationEvents<T> {
-    'select': EventBase<T> & { annotation: Annotation; overlay: Overlay };
-    'deselect': EventBase<T> & { annotation: Annotation; overlay: Overlay };
-    'drag-start': EventBase<T> & { annotation: Annotation };
-    'dragging': EventBase<T> & { annotation: Annotation; coordinate: Coordinate };
-    'drag-end': EventBase<T> & { annotation: Annotation };
```

Comment:
Those turned out to be `Map` events related to an `Annotation`, and not meant to be used on the `Annotation` itself.

```ts
map.addEventListner('drag-start', event => {
  event.annoation // good
});

annotation.addEventListner('drag-start', event => {
  event.annoation // bad
});
```